### PR TITLE
Addressing 2n WG LC comments

### LINKED ISSUE #293: RFC8776-bis DOWNREFs
draft-ietf-pce-sid-algo is used for the definition of the path-metric-delay-minimum identity

This reference in YANG model looks normative

However, the concerns about this dependency seems valid also because there are many I-Ds which are depending from RFC8776-bis

Can we deviate from the SHOULD in RFC 8407 with the objective to avoid this dependency?

---

  ** Downref: Normative reference to an Informational RFC: RFC 2697

  ** Downref: Normative reference to an Informational RFC: RFC 2698

They are referenced for the definition of the identities used to define the corresponding bandwidth profile types

These references in the YANG model look normative.

Unless there are better normative definitions, this is one of the case where we can justify a DOWNREF or deviate from the SHOULD in RFC 8407

---

  ** Downref: Normative reference to an Informational RFC: RFC 2702

It is referenced for the definition of the resource affinities and, with only one exception, with RFC3209 (which is a Standard Track RFC)

IMHO, we can be consistent and always reference RFC 2702 together with RFC 3209, where RFC3209 is a normative reference and RFC2702 is an informative additional reference

---

  ** Downref: Normative reference to an Informational RFC: RFC 4115

I think we can remove any YANG statement referencing RFC 4115 together with the reference. Vendor-specific identities can be defined by those who are implementing RFC 4115 bandwidth profiles

---

  ** Downref: Normative reference to an Experimental RFC: RFC 4125

  ** Downref: Normative reference to an Experimental RFC: RFC 4126

  ** Downref: Normative reference to an Experimental RFC: RFC 4127

These RFCs are used to define some bc-model-type derived identifies, e.g., bc-model-nam identity (Maximum Allocation Bandwidth Constraints Model type)

These references in the YANG model look normative.

It would be good to know why these RFCs are still Experimental.

However, it is worth noting that the corresponding YANG definitions already existed in RFC 8776 but these references were listed as Informative

Can we deviate from the SHOULD in RFC 8407 to keep consistency with RFC 8776?

---

  ** Downref: Normative reference to an Informational RFC: RFC 4427

I think we should expand the WTR acronym to be Wait To Restore (WTR)

The references to RFC 4427 in the YANG model look normative.

The intention here was to reference something generic and independent from a specific protocol (RFC 6378 applies only to MPLS-TP)

ITU-T G.808.1 is a possible alternative but there was some preference for an IETF reference when possible (the Exercise command was the only exception)

Should we add normative references to RFC 6378 (and maybe also RFC 7271) to make RFC 4427 just informative?

---

  ** Downref: Normative reference to an Informational RFC: RFC 4657

It is referenced for the definition of all the identities derived from the lsp-metric-type identity.

These references in the YANG model look normative.

Unless there are better normative definitions, this is one of the case where we can justify a DOWNREF or deviate from the SHOULD in RFC 8407

It is also referenced, together with RFC 9522, for the definition of the identity path-externally-queried (for path computation method).

It might not be an issue RFC 4657 or RFC 9522 are considered a justified DOWNREF or a deviation from the SHOULD in RFC 8407

---

  ** Downref: Normative reference to an Informational RFC: RFC 4736

It is referenced for the definition of the path-reevaluation-request identity

This reference in the YANG model looks normative.

Unless there are better normative definitions, this is one of the case where we can justify a DOWNREF or deviate from the SHOULD in RFC 8407

---

  ** Downref: Normative reference to an Informational RFC: RFC 7823

It is always referenced together with RFC 7471 and RFC 8570.

It can be moved to informative.

---

  ** Downref: Normative reference to an Informational RFC: RFC 9522

It is referenced for the definition of the identities used to define the path-locally-computed, path-externally-queried, path-explicitly-defined path computation methods as well as for the te-optimization-criterion base identity

These references in the YANG model look normative.

Unless there are better normative definitions, this is one of the case where we can justify a DOWNREF or deviate from the SHOULD in RFC 8407

### PR DESCRIPTION
**Pull request by italobusi**
Fixed idnits: fix #293 

NOTE: A diff with the latest published I-D is available at this URL:

https://author-tools.ietf.org/diff?doc_1=draft-ietf-teas-rfc8776-update&url_2=https://raw.githubusercontent.com/italobusi/te/refs/heads/8776-bis-2nd-wg-lc/drafts/te-types-update/draft-ietf-teas-rfc8776-update.txt

### CODE REVIEW

File: drafts/te-types-update/diffs/te-pkt-types/model-diff-spaces.txt [hunk 1]
```diff
@@ -178,139 +168,135 @@
...
-    69c214
-    <           "Bandwidth is explicitly specified.";
-    ---
+    >         description
     >           "Bandwidth profile is explicitly specified.";
-    91c236
+    91c226
     <        MPLS Traffic Engineering";
     ---
     >                  MPLS Traffic Engineering";
```
Comment (tsaad-dev) [inline drafts/te-types-update/diffs/te-pkt-types/model-diff-spaces.txt:186]: Are you sure the text is aligned OK?

File: drafts/te-types-update/diffs/te-pkt-types/model-diff-spaces.txt [hunk 2]
```diff
@@ -178,139 +168,135 @@
...
-    >                        Using TE Metric Extensions
-    234c383
+    >                        Extensions
+    234c370,373
     <              Extensions";
     ---
-    >                        Extensions";
-    247c396
+    >                        Extensions
+    >              RFC 7823: Performance-Based Path Selection for
```
Comment (tsaad-dev) [inline drafts/te-types-update/diffs/te-pkt-types/model-diff-spaces.txt:221]: is text aligned on left margin?

File: drafts/te-types-update/draft-ietf-teas-rfc8776-update.txt [hunk 3]
```diff
@@ -2606,280 +2612,346 @@ Internet-Draft            TE Common YANG Types            September 2024
...
-       base lsp-protection-state;
-       description
-         "There is a signal fail condition on either the working path
-          or the protection path.";
-       reference
-         "RFC 4427: Recovery (Protection and Restoration) Terminology
-                    for Generalized Multi-Protocol Label Switching
-                    (GMPLS)";
-     }
+       identity lockout-of-protection {
```
Comment (tsaad-dev) [inline drafts/te-types-update/draft-ietf-teas-rfc8776-update.txt:2668]: Are there extra white spaces here (also below)?

File: drafts/te-types-update/draft-ietf-teas-rfc8776-update.txt [hunk 4]
```diff
@@ -8713,1160 +8802,2257 @@ B.1.  TE Types YANG Diffs
...
+ <
+ <   organization
+ <     "IETF Traffic Engineering Architecture and Signaling (TEAS)
+ <      Working Group";
+ <   contact
+ <     "WG Web:   <https://datatracker.ietf.org/wg/teas/>
+ <      WG List:  <mailto:teas@ietf.org>
+ <
+ <      Editor:   Tarek Saad
  <                <mailto:tsaad@juniper.net>
```
Comment (tsaad-dev) [inline drafts/te-types-update/draft-ietf-teas-rfc8776-update.txt:8835]: Please update my email here

File: ietf-te-types.yang [hunk 5]
```diff
@@ -1639,41 +1648,41 @@ module ietf-te-types {
...
-      
-       This identity has been obsoleted: the
-       'restoration-scheme-rerouting' identity SHOULD be used
-       instead.";
-    reference
-      "RFC 4872: RSVP-TE Extensions in Support of End-to-End
-                 Generalized Multi-Protocol Label Switching (GMPLS)
-                 Recovery";
-  }
+    // CHANGE NOTE: The identity lsp-protection-reroute
```
Comment (tsaad-dev) [inline ietf-te-types.yang:1669]: white spaces here?

File: drafts/te-types-update/draft-ietf-teas-rfc8776-update.txt (hunk 4 above)
Comment (italobusi) [reply drafts/te-types-update/draft-ietf-teas-rfc8776-update.txt:8835]: This is from a diff with the YANG model in RFC8776 where the juniper mail address was used. The YANG models in RFC8776-bis are all using the gmail address.

**Comment by italobusi** [comment]
Feedbacks from Med on the mailing list

See: https://mailarchive.ietf.org/arch/msg/teas/wXCfmWoYQsDaqzolzVxPy5SyOTo/

**Comment by italobusi** [comment]
Additional feedbacks from Med on the mailing list:

https://mailarchive.ietf.org/arch/msg/teas/LAGfCboxpeMqD-g11v84Z1zGPYU/

**Pull request merged**